webdev/flareact/webdev.sh

```diff
@@ -104,23 +85,22 @@ fi
 case "$1" in
     init )
         check_tools || exit $?
-        check_azion_framework_adapter || exit $?
 
         update_build_script
         update_deploy_script
         mkdir -p public ;;
 
     build )
+        check_init || exit $?
         check_envvars || exit $?
-        check_azion_framework_adapter || exit $?
         echo "{}" > ./args.json
 
-        azion-framework-adapter build --config ./azion/kv.json || exit $?;;
+        npx --yes azion-framework-adapter@0.2.0 build --config ./azion/kv.json || exit $?;;
```

Comment:
Wouldn't be better to drop the path number?
```suggestion
        npx --yes azion-framework-adapter@0.2 build --config ./azion/kv.json || exit $?;;
```

Reply:
npx creates a cache in local machine (in ~/.npm/_npx) and doesn't check package newer versions. To solve this we have to use `npx PACKAGE@latest` or `npx PACKAGE@x.y.z`. 
If we use without PATCH user could not receive new updates. 
We chose to use x.y.z to have more secure deployments.

References:
- https://github.com/npm/cli/issues/2329
- https://github.com/npm/cli/issues/4108